examples/ray-joblib.py

```diff
@@ -0,0 +1,64 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using Ray with
+joblib backend.
+
+In this example, we optimize a classifier configuration for Iris dataset. Classifiers are from
+scikit-learn. We optimize both the choice of classifier (among SVC and RandomForest) and their
+hyper parameters.
+
+"""
+import logging
+
+import joblib
+import ray
+from ray.util.joblib import register_ray
+import sklearn.datasets
+import sklearn.ensemble
+import sklearn.model_selection
+import sklearn.svm
+
+import optuna
+
+
+# TODO(HideakiImamura): Enable the warning after https://github.com/ray-project/ray/issues/13855.
+ray.init(logging_level=logging.ERROR)
```

Comment:
I disable the warning, which is reported in https://github.com/ray-project/ray/issues/13855, to improve the output appearance.